### ISSUE TITLE
Unable to edit data, context-menu is all grayed-out

### ISSUE THREAD

**Issue by sjenset** [handsontable/handsontable#5669]
### Description
In Chrome Version 71.0.3578.98 (Official Build) (64-bit), after creating a dummy table (or, for that sake, a table with actual data) I am unable to edit any of the cells, and the only option not grayed out in the context-menu is "Undo"

### Steps to reproduce
1. See demo

### Demo
https://codepen.io/anon/pen/zyrQpo?editors=1111

### Your environment
* Handsontable version: 6.2.1 from https://cdn.jsdelivr.net/npm/handsontable/dist/handsontable.full.min.js
* Browser Name and version: Chrome Version 71.0.3578.98 (Official Build) (64-bit)
* Operating System: Windows 10 Pro

**Comment by AMBudnik** [comment]
Sorry @sjenset 

but I wasn't able to get the same result. Did you get any error in the console? Do you use any addons? I do not think that in this case the Pro vs Home edition of Windows 10 matters.

![3gif](https://user-images.githubusercontent.com/10757813/50007101-149d9a00-ffb0-11e8-9e16-a2da6869a9ec.gif)

**Comment by sjenset** [comment]
Weird.

No console errors in codepen:

![image](https://user-images.githubusercontent.com/12337525/50007710-14060300-ffb2-11e8-8b46-3787ffb7d00b.png)

No console errors in inspector:

![image](https://user-images.githubusercontent.com/12337525/50007726-1ec09800-ffb2-11e8-87b0-6c455cccf7df.png)

No active Chrome extensions:

![image](https://user-images.githubusercontent.com/12337525/50007750-3009a480-ffb2-11e8-8411-27253cb9d024.png)

**Comment by sjenset** [comment]
Yep

**Comment by sjenset** [comment]
After clicking various cells (left-clicking as well as right-clicking to bring up the context-menu), the only info in the window on the right are the initial two lines:

0 @0.045 [afterCreateRow] 5, 1, "auto",
1 @0.208 [afterChange] null, "loadData",

**Comment by budnix** [comment]
@sjenset What type of PC do you have? I've some assumptions that this can be related to this issue https://github.com/handsontable/handsontable/issues/5586. That issue changed behavior when the touch and regular mouse events are registered. Since v6.2.1 `mousedown` and `mouseup` events are used only on non-touchable devices.

Did your PC support both input methods, such as MS Surface?

**Comment by AMBudnik** [comment]
So the context menu is triggered correctly but does not appear. Can you check if you have an element with className `htContextMenu` in the DOM? 
For the editor, you should get `handsontableInputHolder` className element. 
Maybe the elements are shifted or hidden.

**Comment by sjenset** [comment]
Oh, it appears, it's just "disabled"

![image](https://user-images.githubusercontent.com/12337525/50009754-0489b880-ffb8-11e8-9b05-e2b2bc5e96ac.png)

**Comment by sjenset** [comment]
> @sjenset What type of PC do you have? I've some assumptions that this can be related to this issue #5586. That issue changed behavior when the touch and regular mouse events are registered. Since v6.2.1 `mousedown` and `mouseup` events are used only on non-touchable devices.
> 
> Did your PC support both input methods, such as MS Surface?

Yes, I am on a (docked) laptop which supports touch (on the screen, which is closed while it is in the dock), and I am using a regular, corded mouse for navigation :)

**Comment by AMBudnik** [comment]
So the mystery is partially solved. I hope that we'll be able to fix it in no time.

Thank you for sharing and for the quick updates @sjenset

**Comment by sjenset** [comment]
Can confirm that cell-selection works when using the touch-screen on the device, great work tracking this one down :)

**Comment by budnix** [comment]
Ok, I've prepared a patch and built a bundle. @sjenset Can I ask you to test this [fiddle](https://jsfiddle.net/w0n46u7L/2/)? I have felt that this demo should work for you. It should work for both methods, using a touchscreen and with a plugged mouse.

**Comment by budnix** [comment]
Wonderfull! 👍 Thanks for your prompt reply.
